docs/pages/guides/devtools.mdx

```diff
@@ -0,0 +1,63 @@
+---
+meta:
+  title: "Liveblocks DevTools"
+  parentTitle: "Guides"
+  description:
+    "How to leverage Liveblocks new DevTools to understand and troubleshoot your
+    application fully."
+---
+
+## Why we built a Devtool
+
+Our customers turn to us to expedite the creation of collaborative online
+experiences. Leveraging Liveblocks APIs is powerful, but because we handle the
+resolution and storage of data structures, understanding what is happening
+behind the scenes can be confusing for a new user.
+
+The hope in creating the Liveblocks DevTools was to empower our community so
+that beginners can easily understand and learn Liveblocks. At the same time,
+power users can leverage the tool to troubleshoot, debug, and optimize their
+implementation.
+
+## How the DevTools works
+
+Our DevTools panel is a React + Tailwind app that we can run in the dev tools
+extension tab. We built a basic handshake and state synchronization protocol
+using browser extension message tunneling and making the `@liveblocks/core`
+project aware of it in dev mode. It supports one-way data syncs from the
+Liveblocks client. Much like React and Redux Devtools, the code is compiled away
+in production.
+
+(Could insert image here from notion doc)
+
+## DevTools installation
+
+[Our DevTools](https://chrome.google.com/webstore/detail/liveblocks-devtools/iiagocfmmhknpdalddkbiejnfmbmlffk?hl=en)
+is available from the chrome web store. You can download it from the store, or
+search for it on the main landing page. It is compatible with Chrome and
+Firefox.
+
+## Testing the DevTools with a Liveblocks example
+
+All of our examples hosted on Liveblocks.io are available for you to run locally
+from the [Liveblocks repository](https://github.com/liveblocks/liveblocks) under
+the examples directory.
+
+For the purpose of testing and reviewing how to use the DevTools we will use the
+[Advanced Collaborative Spreadsheet](https://liveblocks.io/examples/collaborative-spreadsheet-advanced/nextjs).
+
+After cloning the GitHub repository and navigating to the right example folder:
+
+```
+git clone https://github.com/liveblocks/liveblocks.git
+
+cd example/nextjs-nextjs-spreadsheet-advanced/
+```
+
+We will need to install packages and create an `.env.local` file, just like we
+would do if we were working on the main repository. First, run
+`bash npm install` and then create the `.env.local` file with your
```

Comment:
@taylorpwhipp are we sure about the bash? isn't it just `npm install`? It's the first time we introduce `bash`, everywhere it's just `npm install`